spa/vite.config.ts

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+
+/**
+ * Documentation: https://vitejs.dev/config/
+ */
+export default defineConfig({
+	plugins: [react()],
+	base: "/spa"
```

Comment:
Shall we use something like `/app` instead of `/spa`, as `/spa` is quite technical details, and `/app` is more generic.